### ISSUE TITLE
Slider not showing intermediate response options

### ISSUE THREAD

**Issue by danielmlow** [ReproNim/reproschema-ui#253]
There is an activity where all items were set to radio. The slider looks much nicer. 

However, when I try setting one of the items to slider, only the first and last response options appear. 

https://github.com/danielmlow/reproschema-library/blob/master/activities/SRIS-SF/items/SRIS-SF_1

![image](https://user-images.githubusercontent.com/14787791/134975554-7300c9a2-7a4a-486f-88b5-47e56922b41c.png)

You can see the value constraints have names for all levels:
https://github.com/danielmlow/reproschema-library/blob/master/activities/SRIS-SF/SRIS-SFvalueConstraints

**Comment by sanuann** [comment]
this is intentionally done this way. the labels for each of the points could be a long(ish) string and would perhaps look clumsy if all are mentioned by default.
@satra should all the labels be present by default? or dependent on anything?

**Comment by satra** [comment]
you could make it a ui parameter potentially. but i agree in most cases sticking all the labels will make the cleanliness go away.

**Comment by danielmlow** [comment]
Yeah, let's make a ui parameter. Can't people add `<br>` to the labels so it appears on two rows?

**Comment by sanuann** [comment]
how about placing the labels in the tooltip?

**Comment by danielmlow** [comment]
What's a tooltip? 

Would <br> work? Then I can place the labels on 2 rows so they don't overlap horizontly.

**Comment by satra** [comment]
please do consider a phone screen when testing. a number of options that may look good on a computer don't work well on a phone.

**Comment by sanuann** [comment]
> please do consider a phone screen when testing. a number of options that may look good on a computer don't work well on a phone.

good point. thanks for that reminder.

**Comment by sanuann** [comment]
> What's a tooltip?
> 
> Would work? Then I can place the labels on 2 rows so they don't overlap horizontly.

i'll send you a screenshot in some time

**Comment by sanuann** [comment]
also, I just had a thought - if you want to show all the labels then wouldn't it be better to show the question as a radio item?
How does one decide when to have a radio vs slider?

**Comment by danielmlow** [comment]
Yes, we're currently using radio option. But multiple people said slider looks a lot nicer and than switching between slider and radio can be confusing. If we can put labels on two or more rows with <br> than it should work. 

Placing on the tooltip could work, but the labels may still be long and overlap without <br>

**Comment by satra** [comment]
@danielmlow - are they doing it on a phone? tooltips are a little difficult and require people to actually hover. so that may not be a useful tool in this context.  given the different viewports of devices, if one wants labels on every point, i think the radio button may be the best option.

**Comment by sanuann** [comment]
> @danielmlow - are they doing it on a phone? tooltips are a little difficult and require people to actually hover. so that may not be a useful tool in this context. given the different viewports of devices, if one wants labels on every point, I think the radio button may be the best option.

I second this.

**Comment by sanuann** [comment]
or maybe we can have another label at the midpoint.

**Comment by danielmlow** [comment]
That's fair, I guess we'll do radio for now. 

Sanu, some questionnaires have labels for all responses, so we can't choose what labels to use, we need to use them all to match the questionnaire.
